libs/user/website/src/lib/personal-data-form/personal-data-form-container.tsx

```diff
@@ -20,39 +19,42 @@ export function PersonalDataFormContainer({className, mediaEmail}: PersonalDataF
 
   const [uploadImage] = useUploadImageMutation()
   const [updatePassword] = useUpdatePasswordMutation()
-  const [updateUser, updateUserData] = useUpdateUserMutation()
+  const [updateUser] = useUpdateUserMutation()
 
-  const handleOnImageUpload = (input: ChangeEvent<HTMLInputElement> | null) => {
-    uploadImage({
+  const handleOnImageUpload = async (input: ChangeEvent<HTMLInputElement> | null) => {
+    await uploadImage({
       variables: {
-        uploadImageInput: {file: input?.target?.files![0] as File} || null
+        uploadImageInput: input ? {file: input.target?.files![0] as File} : null
```

Comment:
deleting didn't work because `uploadImageInput` has to be null. uploadImageInput was never null in the previous implementation